frida_tools/tracer.py

```diff
@@ -345,15 +345,35 @@ def _create_resolver_script(self):
     return workingSet;
 }
 
+function parseExportsFunctionPattern(pattern) {
+    var res = pattern.split('!');
+    var m, f;
+    
+    if (res.length === 1) {
+        m = '*';
+        f = res[0];
+    } else {
+        m = (res[0] === '') ? '*' : res[0];
+        f = (res[1] === '') ? '*' : res[1];
+    }
+    
+    return {
+        module: m,
+        function: f
+    };
+}
+
 function includeFunction(pattern, workingSet) {
-    moduleResolver().enumerateMatches('exports:*!' + pattern).forEach(function (m) {
+    var obj = parseExportsFunctionPattern(pattern);
+    moduleResolver().enumerateMatches('exports:' + obj['module'] + '!' + obj['function']).forEach(function (m) {
```

Comment:
```suggestion
    moduleResolver().enumerateMatches('exports:' + obj.module + '!' + obj.function).forEach(function (m) {
```

(A bit more readable this way.)

Reply:
Good point, done.